src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper implements DtoResponseMapper<ShoppingCartResponseDto,
+        ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto shoppingCartResponseDto = new ShoppingCartResponseDto();
+        shoppingCartResponseDto.setId(shoppingCart.getId());
+        shoppingCartResponseDto.setUserId(shoppingCart.getId());
```

Comment:
Aren't they the same?

Reply:
I think better pass difference id variable for user and shopping cart.